src/resmom/mom_main.c

```diff
@@ -7937,6 +7947,11 @@ main(int argc, char *argv[])
 	char			path_hooks_rescdef[MAXPATHLEN+1];
 
 #ifdef WIN32
+
+#ifdef PYTHON
+	PyObject		*path;
+	char			buf[MAXPATHLEN];
```

Comment:
My use of 'buf' in the code is actually for that one path that I'm appending to the Python object 'path'. So I don't need to make it larger since I'm only dealing with 1 path to add at a time.